addons/purchase/security/ir.model.access.csv

```diff
@@ -19,8 +19,10 @@ access_res_partner_purchase_user,res.partner purchase,base.model_res_partner,gro
 access_account_journal,account.journal,account.model_account_journal,group_purchase_user,1,0,0,0
 access_account_journal_manager,account.journal,account.model_account_journal,group_purchase_manager,1,0,0,0
 access_account_move,account.move,account.model_account_move,group_purchase_user,1,0,0,0
-access_account_move_line,account.move.line,account.model_account_move_line,group_purchase_user,1,0,0,0
+access_account_move_line,account.move.line,account.model_account_move_line,group_purchase_user,1,1,1,0

```

Comment:
I might be missing something, but why is that necessary on `account.move.line` and not on `account.move`?

Reply:
I did not realize that there are two lines for account.move in this file. I remove the stricter one

Reply:
Good catch